core/src/main/java/io/questdb/cairo/wal/seq/TableSequencerImpl.java

```diff
@@ -267,8 +266,12 @@ private void applyToMetadata(TableMetadataChange change) {
         try {
             change.apply(sequencerMetadataUpdater, true);
             metadata.syncToMetaFile();
-        } catch (SqlException e) {
-            throw CairoException.critical(0).put("error applying alter command to sequencer metadata [error=").put(e.getFlyweightMessage()).put(']');
+        } catch (CairoException e) {
+            int errno = e.getErrno();
+            LOG.error().$("cannot apply structure change from WAL to table [table=").utf8(sequencerMetadataUpdater.getTableName())
```

Comment:
this should be "could not apply..." as in "we tried, but it failed"

"cannot apply" means "we will not apply, because stars did not align"

Reply:
Renamed in 0da96063674df9fb51bacc334a93b9d4971f1a16